hpx/components/component_storage/component_storage.hpp

```diff
@@ -26,6 +26,11 @@ namespace hpx { namespace components
     public:
         component_storage(hpx::id_type target_locality);
 
+        template <typename ... Ts>
+        component_storage(Ts && ... ts)
+          : base_type(std::forward<Ts>(ts)...)
+        {}
+
```

Comment:
I can see that, but what do we need it for? Wouldn't it make the other constructor obsolete or ambiguous? Shouldn't we rather introduce constructors that this class absolutely need?